dig_test.go

```diff
@@ -543,6 +598,49 @@ func TestEndToEndSuccess(t *testing.T) {
 		}), "invoke should succeed, pulling out two named instances")
 	})
 
+	t.Run("name tags on result structs recurse", func(t *testing.T) {
```

Comment:
Errr...is "recurse" a verb? I always thought it was "recur" 🤷‍♂️

Reply:
https://en.wiktionary.org/wiki/recurse#Verb

:man_shrugging: